src/test/java/org/wahlzeit_revisited/repository/PhotoRepositoryIT.java

```diff
@@ -0,0 +1,124 @@
+package org.wahlzeit_revisited.repository;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.wahlzeit_revisited.BaseModelTest;
+import org.wahlzeit_revisited.model.*;
+
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+
+public class PhotoRepositoryIT extends BaseModelTest {
```

Comment:
I believe `IT` means integration test in that context?

Great example for a test that you don't want to run in the usual unit test phase => good example for ADAP test suite exercise!

Reply:
`IT` stands for integration test, yes